bin/start

```diff
@@ -30,6 +30,7 @@ usage() {
                                start from genesis (no data) [default]
             -p --persist       start with persistent data
             -b --build         (re)build docker images before starting
+            --prod             start NEXT with SSL configuration
```

Comment:
is this supposed to be -b --prod?  or just --prod?  Do we have a single letter execution?

Reply:
You would pass in the --prod flag alone. If you would like to build, you would add -b as well. I didn't add a single letter execution equivalent for the --prod flag. I can if that is best practice.